frontend/src/components/Forum/Forum.js

```diff
@@ -0,0 +1,99 @@
+import FilterThreads from './FilterThreads'
+import SearchThreads from './SearchThreads'
+import CardView from './CardView'
+import CreateThreadButton from '../CreateThread/CreateThreadButton'
+import ListView from './ListView'
+import { Breadcrumb, Layout, Pagination, Menu } from 'antd'
+import { AppstoreOutlined, BarsOutlined, EnvironmentOutlined } from '@ant-design/icons'
+import '../../style/Forum.css'
+import { useSelector } from 'react-redux'
+import { useState } from 'react'
+import { threads } from '../../mocks/forumMock'
+
+const { Content } = Layout
+
+function Forum (props) {
+  //reference: ChatGPT (separate list of items into different pages)
```

Comment:
please remove unnecessary comments

Reply:
I have removed the ChatGPT reference part and what I considered to be unnecessary commented code.